utils/README.md

```diff
@@ -0,0 +1,7 @@
+# Utility Files
```

Comment:
Where does this `utils` folder come from? I seems like external code dropped in the project. I don't think it should be included in the repo. We should probably make the setup.py fetch this code as part of the install or have something like a "fetch_dash_utils" method, if available at a reliable source.

Reply:
correct, it is from the mne-dash example we initially looked at. it should be removed.